lib/AST/FeatureSet.cpp

```diff
@@ -693,6 +693,16 @@ static bool usesFeatureConformanceSuppression(Decl *decl) {
   return false;
 }
 
+static bool usesFeatureBitwiseCopyable2(Decl *decl) {
```

Comment:
It would be helpful to have a `ModuleInterface` test that shows the effect this has on interface printing.

Reply:
Added a check that the decls are in the stdlib and two test cases, one stdlib, one not.